lib/simplifyExpression/simplify.js

```diff
@@ -23,14 +20,7 @@ function simplify(node, debug=false) {
     // removing parens isn't counted as a step, so try it here
     simplifiedNode = removeUnnecessaryParens(flattenOperands(node), true);
   }
-  // unflatten the node.
-  return unflatten(simplifiedNode);
-}
-
-// Unflattens a node so it is in the math.js style, by printing and parsing it
-// again
-function unflatten(node) {
```

Comment:
Let's keep flatten around until we port everything over and then we can start experimenting.